assets/sass/okta-sign-in.scss

```diff
@@ -1,32 +1,42 @@
 @import 'compass/css3';
 
 @import 'widgets/jquery.qtip';
```

Comment:
Where does this come from? And does it drop styles into the page or is it just a helper file?